services/web-app/pages/designing/get-started.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: Get started
+description:
+---
```

Comment:
is this supp to be empty? should we add a "coming soon" or something ?

Reply:
I can add add coming soon, this is what Andrea is working on here https://github.com/carbon-design-system/carbon-platform/pull/784